src/main/webapp/app/course/competencies/competency-management/competency-management-table.component.html

```diff
@@ -0,0 +1,116 @@
+        <div ngbDropdown class="d-inline">
+            <button id="importCompetenciesDropdown" class="btn btn-primary" ngbDropdownToggle>
+                <fa-icon [icon]="faFileImport" />
+                {{ 'artemisApp.' + competencyType + '.manage.importButton' | artemisTranslate }}
+            </button>
+            <div ngbDropdownMenu aria-labelledby="importCompetenciesDropdown">
+                <button class="btn" id="competencyImportAllButton" (click)="openImportAllModal()" ngbDropdownItem>
+                    <fa-icon [icon]="faFileImport" />
+                    <span [jhiTranslate]="'artemisApp.' + competencyType + '.manage.importAllButton'"></span>
+                </button>
+                <a class="btn" [routerLink]="['/course-management', courseId, competencyType + '-management', 'import']" ngbDropdownItem>
+                    <fa-icon [icon]="faFileImport" />
+                    <span [jhiTranslate]="'artemisApp.' + competencyType + '.manage.importFromCoursesButton'"></span>
+                </a>
+                @if (standardizedCompetenciesEnabled) {
+                    <a class="btn" [routerLink]="['/course-management', courseId, competencyType + '-management', 'import-standardized']" ngbDropdownItem>
+                        <fa-icon [icon]="faFileImport" />
+                        <span [jhiTranslate]="'artemisApp.' + competencyType + '.manage.importStandardizedButton'"></span>
+                    </a>
+                }
+            </div>
+        </div>
+        <a class="btn btn-primary" [routerLink]="['/course-management', courseId, competencyType + '-management', 'create']">
+            <fa-icon [icon]="faPlus" />
+            <span [jhiTranslate]="'artemisApp.' + competencyType + '.manage.createButton'"></span>
+        </a>
+    </div>
+</div>
+<div class="container-fluid overflow-auto">
+    @if (courseCompetencies.length) {
+        <table class="table table-striped">
+            <thead>
+                <tr>
+                    <th class="col">
+                        <span jhiTranslate="global.field.id"></span>
+                    </th>
+                    <th class="col">
+                        <span jhiTranslate="artemisApp.competency.title"></span>
+                    </th>
+                    <th class="col">
+                        <span jhiTranslate="artemisApp.competency.description"></span>
+                    </th>
+                    <th class="col">
+                        <span jhiTranslate="artemisApp.competency.taxonomy"></span>
+                    </th>
+                    <th class="d-none col-lg d-lg-table-cell">
+                        <span jhiTranslate="artemisApp.competency.softDueDate"></span>
+                    </th>
+                    <th class="d-none col-lg d-lg-table-cell">
+                        <span jhiTranslate="artemisApp.competency.masteredStudents"></span>
+                    </th>
+                    <th class="col">
```

Comment:
we dont want this to behave like the other two headers above? Like d-none is not needed but the other classes?